fem/fe.hpp

```diff
@@ -1958,13 +1975,20 @@ class Poly_1D
       int etype;
       DenseMatrixInverse Ai;
       mutable Vector x, w;
+      // The following data members are used for "integrated basis type", which
+      // is defined in terms of nodal basis of one degree higher.
+      mutable Vector u_aux, d_aux, d2_aux;
+      Basis *auxiliary_basis; // Non-NULL only for etype == Integrated
 
    public:
       /// Create a nodal or positive (Bernstein) basis
       Basis(const int p, const double *nodes, EvalType etype = Barycentric);
       void Eval(const double x, Vector &u) const;
       void Eval(const double x, Vector &u, Vector &d) const;
       void Eval(const double x, Vector &u, Vector &d, Vector &d2) const;
+      void EvalIntegrated(const Vector &d, Vector &i) const;
```

Comment:
The `Eval` functions could use some brief documentation, at least to describe the arguments.

Reply:
I added a Doxygen comment here explaining what it is doing.